src/linker/cli.py

```diff
@@ -114,7 +113,7 @@ def run(
     type=click.Path(exists=True, resolve_path=True),
 )
 @click.argument(
-    "log_dir",
+    "diag_dir",
```

Comment:
Nit: I don't like abbreviating this. I think just doing `diagnostics` is fine. At first I thought you were meaning dag.

Reply:
I almost said the same!